samples/core/guide/autograph.ipynb

```diff
@@ -415,14 +417,14 @@
       "cell_type": "code",
       "source": [
         "@autograph.convert()\n",
-        "def f(x):\n",
-        "  assert x != 0, 'Do not pass zero!'\n",
-        "  return x * x\n",
+        "def inverse(x):\n",
+        "  assert x != 0.0, 'Do not pass zero!'\n",
+        "  return 1.0/x\n",
         "\n",
         "with tf.Graph().as_default():  \n",
         "  with tf.Session():\n",
         "    try:\n",
-        "      print(f(tf.constant(0)).eval())\n",
+        "      print(inverse(tf.constant(0.0)).eval())\n",
```

Comment:
could you use sess.run(inverse(...)) instead of .eval() here and elsewhere?

Reply:
done